package.json

```diff
@@ -28,7 +29,8 @@
   },
   "scripts": {
     "dev": "next dev",
-    "dev:docker": "ncp ./.env.docker ./.env && next dev",
+    "predev:docker": "ncp ./.env.docker ./.env",
+    "dev:docker": "next dev",
```

Comment:
Not sure what these are for but should be in a separate PR if this PR is for updating deps

Reply:
Fixed 👍